pkg/util/util.go

```diff
@@ -298,3 +301,61 @@ func MustNewRequirement(key string, op selection.Operator, vals []string) *label
 	}
 	return r
 }
+
+func BuildAdditionalVolumeAndVolumeMount(tc *v1alpha1.TidbCluster, memberType v1alpha1.MemberType) ([]corev1.VolumeMount, []corev1.PersistentVolumeClaim) {
```

Comment:
Would it be better to pass `storageVolumes` and `storageClassName` by params.
It seems not easy to add support for new Component(need to update `BuildAdditionalVolumeAndVolumeMount`)

Reply:
Ok.